examples/acados_python/pendulum_on_cart/ocp/test_casadi_formulation.py

```diff
@@ -0,0 +1,63 @@
+
+import sys
+sys.path.insert(0, '../common')
+
+import numpy as np
+import casadi as ca
+from typing import Union
+
+from acados_template import AcadosOcp, AcadosOcpSolver, AcadosCasadiOcpSolver
+from ocp_example_cost_formulations import formulate_ocp, T_HORIZON
+
+from utils import plot_pendulum
+
+def get_x_u_traj(ocp_solver: Union[AcadosOcpSolver, AcadosCasadiOcpSolver], N_horizon: int):
```

Comment:
Not sure if this is really needed, why not use `store_iterate_to_object`? Maybe extend this function with an extra flag `primal_variables_only` or something like that

Reply:
Implementing store/load iterate functions is planned, see https://github.com/acados/acados/issues/1525
For now, I would keep this. But indeed all equivalence checks should be done via iterates soon!